code/unclassified/src/Max_of_array/Max_of_array.cpp

```diff
@@ -0,0 +1,27 @@
+ #include <iostream>
+using namespace std;
```

Comment:
This line is bad practice. 
https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice